docs/DeveloperGuide.adoc

```diff
@@ -782,13 +782,16 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+* tech-savvy individual
```

Comment:
Perhaps it will be better to state "tech-savvy veterinary practioners"

Reply:
Agreed with @Aquarinte; I think "tech-savvy" veterinary practitioners" will be better, since it will make it very clear that we are targeting vets.

Reply:
ok, will change